internal/stack/boot.go

```diff
@@ -123,21 +118,19 @@ func dockerComposeUp(daemonMode bool, stackVersion string) error {
 		return errors.Wrap(err, "locating stack directory failed")
 	}
 
-	args := []string{
-		"-f", filepath.Join(stackDir, "snapshot.yml"),
-		"up",
+	c, err := compose.NewProject(dockerComposeProjectName, filepath.Join(stackDir, "snapshot.yml"))
+	if err != nil {
+		return errors.Wrap(err, "could not create docker compose project")
```

Comment:
(nit, just note about polishing)

It would be cool to unify naming: docker compose vs docker-compose vs Docker Compose (not sure which is, officially, the correct one).

Reply:
Looking at https://docs.docker.com/compose/, it appears the name of the project is "Docker Compose". The CLI tool is called `docker-compose`. Since we are referring more to the project here, I will update all references to use "Docker Compose".

Reply:
Updated in 0882065.